en/lessons/libraries/nimble-publisher.md

```diff
@@ -221,7 +221,13 @@ end
 
 ### View
 
-Create the view module where you can place the helpers needed to render the view. By now it's just:
+Create the view module where you can place the helpers needed to render the view. First create the file: 
+
```

Comment:
We could consolidate this wording a little to something like:

`Create a file, lib/nimble_school_web/views/blog_view.ex, and add the following content:`

Reply:
Yes, I prefer this wording as the important thing is to create the file, rather than how to create it - it's not important if you `touch` it via shell or just -like me- dare to use a mouse to do it :D